run.sh

```diff
@@ -1,15 +1,57 @@
 #!/bin/bash
 set -e
 set -x
-export DEVPI_SERVERDIR=/mnt
-export DEVPI_CLIENTDIR=/tmp/devpi-client
-[[ -f $DEVPI_SERVERDIR/.serverversion ]] || initialize=yes
-
-devpi-server --start --host 0.0.0.0 --port 3141
-if [[ $initialize = yes ]]; then
-  devpi use http://localhost:3141
-  devpi login root --password=''
-  devpi user -m root password="${DEVPI_PASSWORD}"
-  devpi index -y -c public pypi_whitelist='*'
-fi
-tail -f $DEVPI_SERVERDIR/.xproc/devpi-server/xprocess.log
+
+# Default environment if none is provided
+DEVPI_SERVERIP=${DEVPI_SERVERIP:-127.0.0.1}
+DEVPI_SERVERPORT=${DEVPI_SERVERPORT:-3143}
+DEVPI_SERVERDIR=${DEVPI_SERVERDIR:-/mnt}
+DEVPI_CLIENTDIR=${DEVPI_CLIENTDIR:-/tmp/devpi-client}
+DEVPI_CLIENTPASSWORD=${DEVPI_CLIENTPASSWORD:-""}
+DEVPI_LDAPCONFILE=${DEVPI_LDAPCONFILE:-""}
+
+NGINX_IP=${NGINX_IP:-0.0.0.0}
+NGINX_PORT=${NGINX_PORT:-3141}
+NGINX_ENABLETLS=${NGINX_ENABLETLS:-false}
+NGINX_TLSPORT=${NGINX_TLSPORT:-3142}
+NGINX_TLSPATH=${NGINX_TLSPATH:-/opt/nginx/certs/default}
+NGINX_TLSCERT=${NGINX_TLSCERT:-devpi.crt}
+NGINX_TLSKEY=${NGINX_KEY:-devpi.key}
```

Comment:
Oh, just a mistake. I didn't know this syntax, so I googled it and copy/paste from stackoverflow where it was slightly different. Will change for `: ${VAR:=value}`